webpack/assets/javascripts/react_app/components/charts/ChartHeader.jsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import Component from '../common/BindingComponent';
+
+export default class extends Component {
+    constructor(props) {
+        super(props);
+        this.bind(
+            'onClick'
+        );
+    }
+
+    onClick() {
+        this.props.onClick();
+    }
+
+    render() {
+        // TODO: refactor to handle tooltip and placement here or in chart box
+        return (
+            <h4 className={this.props.cssClass}
```

Comment:
hmm.. you might be right, but I think a secondary goal of us is to align with patternfly design, so maybe we should revisit our current style, additionally, i believe our component should support patternfly common usage cases regardless.